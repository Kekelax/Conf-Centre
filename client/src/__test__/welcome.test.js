import React from "react";
import Welcome from "../components/welcome";
import renderer from "react-test-renderer";

it("Welcome page renders correctly", () => {
  const tree = renderer.create(<Welcome />).toJSON();
  expect(tree).toMatchSnapshot();
});
