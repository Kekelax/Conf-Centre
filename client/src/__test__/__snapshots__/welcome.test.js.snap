// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome page renders correctly 1`] = `
<div
  className="welcomeWrapper"
>
  <div
    className="welcomeContainer"
  >
    <div
      className="welcomeHeader"
    >
      <h1>
        Welcome
      </h1>
      <br />
      <br />
    </div>
    <div
      className="welcomeSubtext"
    >
      <p>
        Please login or register to view the dashboard. 
        <br />
      </p>
    </div>
    <row>
      <a
        aria-describedby={null}
        className=""
        href="/login"
        onBlur={[Function]}
        onFocus={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        title="Go to the login page"
      >
        <button
          className="welcomebtn orangeButton"
        >
          Login
        </button>
      </a>
      <a
        aria-describedby={null}
        className=""
        href="/sign-up"
        onBlur={[Function]}
        onFocus={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        title="Go to the sign-up page"
      >
        <button
          className="welcomebtn orangeButton"
        >
          Register
        </button>
      </a>
    </row>
  </div>
</div>
`;
